lib/routes/gov/zj/ningborsjnotice.ts

```diff
@@ -0,0 +1,50 @@
+import { Route } from '@/types';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+
+export const route: Route = {
+    path: '/zj/ningborsjnotice/:colId?',
+    categories: ['government'],
+    example: '/gov/zj/ningborsjnotice/1229676740',
+    parameters: {
+        colId: '公告分类id、详细信息点击源网站http://rsj.ningbo.gov.cn/请求中寻找',
+    },
+    radar: [
+        {
+            source: ['rsj.ningbo.gov.cn/col/col1229676740/index.html'],
+            target: '/zj/ningborsjnotice/:colId?',
+        },
+    ],
+    name: '宁波市人力资源和社会保障局-公告',
+    url: 'rsj.ningbo.gov.cn',
+    maintainers: ['HaoyuLee'],
+    description: `
+    | 公告类别         | colId |
+    | ------------ | -- |
+    | 事业单位进人公告     | 1229676740  |
```

Comment:
The description is still indenting with 4 spaces.
https://github.com/DIYgod/RSSHub/pull/16404#discussion_r1711671191